ni_measurementlink_service/_internal/service_manager.py

```diff
@@ -73,41 +73,48 @@ def start(
             int: The port number of the server
 
         """
-        self.server = grpc.server(
-            logging_pool.pool(max_workers=10),
-            options=[
-                ("grpc.max_receive_message_length", -1),
-                ("grpc.max_send_message_length", -1),
-            ],
-        )
-        servicer_v1 = MeasurementServiceServicerV1(
-            measurement_info,
-            configuration_parameter_list,
-            output_parameter_list,
-            measure_function,
-        )
-        v1_measurement_service_pb2_grpc.add_MeasurementServiceServicer_to_server(
-            servicer_v1, self.server
-        )
-        servicer_v2 = MeasurementServiceServicerV2(
-            measurement_info,
-            configuration_parameter_list,
-            output_parameter_list,
-            measure_function,
-        )
-        v2_measurement_service_pb2_grpc.add_MeasurementServiceServicer_to_server(
-            servicer_v2, self.server
-        )
-        port = str(self.server.add_insecure_port("[::]:0"))
-        self.server.start()
-        _logger.info("Measurement service hosted on port: %s", port)
-        self.discovery_client.register_measurement_service(port, service_info, measurement_info)
-
-        self.port = port
-        return port
+        try:
+            self.server = grpc.server(
+                logging_pool.pool(max_workers=10),
+                options=[
+                    ("grpc.max_receive_message_length", -1),
+                    ("grpc.max_send_message_length", -1),
+                ],
+            )
+            servicer_v1 = MeasurementServiceServicerV1(
+                measurement_info,
+                configuration_parameter_list,
+                output_parameter_list,
+                measure_function,
+            )
+            v1_measurement_service_pb2_grpc.add_MeasurementServiceServicer_to_server(
+                servicer_v1, self.server
+            )
+            servicer_v2 = MeasurementServiceServicerV2(
+                measurement_info,
+                configuration_parameter_list,
+                output_parameter_list,
+                measure_function,
+            )
+            v2_measurement_service_pb2_grpc.add_MeasurementServiceServicer_to_server(
+                servicer_v2, self.server
+            )
+            port = str(self.server.add_insecure_port("[::]:0"))
+            self.server.start()
+            _logger.info("Measurement service hosted on port: %s", port)
+            self.discovery_client.register_measurement_service(port, service_info, measurement_info)
+
+            self.port = port
+
+            return port
+        except Exception as e:
+            raise e
```

Comment:
I don't understand what the try/except blocks in this file are intended to accomplish. You don't need to catch the exception and re-raise it.

Reply:
Removed the `try`/`except` block.